src/config/serde/bitcoin_amount.rs

```diff
@@ -0,0 +1,57 @@
+use crate::bitcoin::*;
+use serde::{Deserialize, Deserializer, Serializer};
+
+// pub fn deserialize<'de, D>(deserializer: D) -> Result<Amount, D::Error>
```

Comment:
❓ That supposed to be still there?

Reply:
oh thanks! Yeah I wanted to try something at some point, I'll try now.